compiler/damlc/daml-compiler/src/DA/Daml/Compiler/Repl.hs

```diff
@@ -154,12 +154,31 @@ topologicalSort lfPkgs = map toPkg $ topSort $ transposeG graph
     toPkg = (\(pkg, _, _) -> pkg) . fromVertex
 
 data ReplState = ReplState
-  { moduleNames :: [LF.ModuleName]
+  { imports :: [ImportDecl GhcPs]
```

Comment:
You probably want bang patterns on this given that you’re using `$!` below.

Reply:
Good catch, done.